src/usr/local/www/services_unbound_domainoverride_edit.php

```diff
@@ -146,6 +146,12 @@ function String_Begins_With($needle, $haystack) {
 	));
 }
 
+$section->setHelp("This page is used to specify domains for which the resolver's standard DNS lookup process will be overridden, " .
```

Comment:
This is still broken.

`Fatal error: Call to undefined method Form_Section::setHelp() in /usr/local/www/services_unbound_domainoverride_edit.php on line 149 Call Stack: 0.0001 228496 1. {main}() /usr/local/www/services_unbound_domainoverride_edit.php:0 PHP ERROR: Type: 1, File: /usr/local/www/services_unbound_domainoverride_edit.php, Line: 149, Message: Call to undefined method Form_Section::setHelp()`